code/game/dna/dna2_helpers.dm

```diff
@@ -145,7 +145,7 @@
 		H.b_eyes   = dna.GetUIValueRange(DNA_UI_EYES_B,    255)
 		H.update_eyes()
 
-		H.s_tone   = 35 - dna.GetUIValueRange(DNA_UI_SKIN_TONE, 220) // Value can be negative.
+		H.s_tone   = dna.GetUIValueRange(DNA_UI_SKIN_TONE, 220)
```

Comment:
What's up with this?

Reply:
Back in the ancient era, skin tone was calculated by doing `35 - x`, where `x` is a value from 30 to 220.